### DISCUSSION TITLE
compose-lints warnings not showing up in assemble/run output

### DISCUSSION THREAD

**Discussion by carstenhag** [slackhq/compose-lints#357] [Q&A]
For some reason I am not seeing any warnings/errors from this library's checks. For example I run `./gradlew lint` and there are dozens of lines like this:

```
w: file:///x/app/src/main/java/com/bmw/chargenow/presentation/views/vehicle/carmakeselection/CarMakeSelectionActivity.kt:149:9 'overridePendingTransition(Int, Int): Unit' is deprecated. Deprecated in Java
```
but I am not seeing any compose warnings.

When I open the lint.xml file I do see them. Also in the IDE, when I have the file open, I see the issue.

**Comment by ZacSweers** [comment]
that example warning is unrelated to lint or compose, that's just a standard kotlin compiler warning. This issue is impossible to advise on with the limited information you've provided, and is also more of a debugging question so I've converted it
